azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/ActiveDirectoryGroup.java

```diff
@@ -34,4 +35,97 @@
      * @return mail field.
      */
     String mail();
+
+    /**
+     * List the members in the group.
+     * @return an unmodifiable set of the members
+     */
+    Set<ActiveDirectoryObject> listMembers();
+
+    /**
+     * List the members in the group.
+     * @return an unmodifiable set of the members
+     */
+    Observable<ActiveDirectoryObject> listMembersAsync();
+
+    /**
+     * Container interface for all the definitions that need to be implemented.
+     */
+    interface Definition extends
+            DefinitionStages.Blank,
+            DefinitionStages.WithEmailAlias,
+            DefinitionStages.WithCreate {
+    }
+
+    /**
+     * Grouping of all the AD group definition stages.
+     */
+    interface DefinitionStages {
+        /**
+         * The first stage of the AD group definition.
+         */
+        interface Blank extends WithEmailAlias {
+        }
+
+        /**
+         * An AD Group definition allowing mail nickname to be specified.
+         */
+        interface WithEmailAlias {
+            WithCreate withEmailAlias(String mailNickname);
+        }
+
+        /**
+         * An AD Group definition allowing members to be added.
+         */
+        interface WithMember {
+            /**
+             * Add a member based on its object id. The member can be a user, a group, a service principal, or an application.
+             * @param objectId the Active Directory object's id
+             * @return the next AD Group definition stage
+             */
+            WithCreate withMember(String objectId);
+        }
+
+        /**
+         * An AD group definition with sufficient inputs to create a new
+         * group in the cloud, but exposing additional optional inputs to
+         * specify.
+         */
+        interface WithCreate extends
+                Creatable<ActiveDirectoryGroup>,
+                WithMember {
+        }
+    }
+
+    /**
+     * Grouping of all the AD group update stages.
+     */
+    interface UpdateStages {
+        /**
+         * An AD Group definition allowing members to be added or removed.
+         */
+        interface WithMember {
+            /**
+             * Adds a member based on its object id. The member can be a user, a group, a service principal, or an application.
+             * @param objectId the Active Directory object's id
+             * @return the next AD Group update stage
+             */
+            Update withMember(String objectId);
```

Comment:
would it make sense to also have overloads that take a specific type, like `ActiveDirectoryUser`, etc?